Projects/Chat/Sources/Navigation/ChatNavigation.swift

```diff
@@ -44,8 +45,16 @@ public class ChatNavigationViewModel: ObservableObject {
     }
 }
 
-public enum ChatRedirectViewType {
+public enum ChatRedirectViewType: Hashable {
     case notification
+    case claimDetail(id: String)
+}
+
+extension ChatRedirectViewType: TrackingViewNameProtocol {
+    public var nameForTracking: String {
+        return "trackingName"
```

Comment:
WE can do the proper naming now.
switch self and place proper data - "ClaimDetailView" or "AskForPushNotifications"

Reply:
Uups, totally forgot about this one :)